src/main/java/bio/terra/app/controller/RepositoryApiController.java

```diff
@@ -4,27 +4,12 @@
 import bio.terra.app.utils.ControllerUtils;
 import bio.terra.controller.RepositoryApi;
 import bio.terra.app.controller.exception.ValidationException;
-import bio.terra.common.exception.InternalServerErrorException;
-import bio.terra.common.exception.UnauthorizedException;
+import bio.terra.model.*;
```

Comment:
We don't use '*' imports. I can show you how to dissuade IntelliJ from generating them.

Reply:
Understood and thanks. Fixed in most recent commit.